### ISSUE TITLE
Elfeed integration

### ISSUE THREAD

**Issue by ch1ebak** [chenyanming/wallabag.el#13]
Any way to add articles straight from elfeed? Pretty much the only thing stopping me from switching from Pocket.

**Comment by chenyanming** [comment]
Use `wallabag-add-entry`

**Comment by ch1ebak** [comment]
when i use wallabag-add-entry the prompt asks me to add an url instead of adding the selected article. pressing enter at the prompt adds an empty article to wallabag

**Comment by chenyanming** [comment]
Currently, you can copy the link from elfreed first them paste into the promot. It is the general way to add article. If you really need to add from elfeed, probaly you can write a simple function to get url from elfeed and pass url to wallabag-add-entry...

**Comment by ch1ebak** [comment]
this workaround worked, thanks!

**Comment by meliache** [comment]
I agree it would be useful having a version of `wallabag-add-entry` where at least the URL can be given as a parameter so that it's easier to wrap it in your own functions. (For example, I would also add it to `embak-url-map` so that it becomes a menu item whenever I run `embark-act` on a URL, etc.)

Also, when it is a function that users should be able to run outside of wallabag, maybe autoload it too? Though I'm fine if that's not done by default, I'll just add it to `:commands` in my `use-package` macro.

**Comment by chenyanming** [comment]
It is supported elfeed-show-mode now.
